<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>111</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(75, 10, 240, 255), stop:1 rgba(255, 0, 0, 255));
border-radius:20px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font :32pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>神笔马良</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;华文细黑&quot;;</string>
     </property>
     <property name="text">
      <string>无需复杂配置，直接登录，进行StableDiffusion绘图</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/icons/icons/document_0.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>371</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_2{
background-color: rgb(255, 255, 255);
border-top-right-radius:20px;
border-bottom-right-radius: 20px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>431</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>301</width>
       <height>371</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>6</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>5</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="stackedWidget_2">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	
	background-color: rgba(255, 255, 255, 0);
	border:none;
	border-bottom:1px solid black;
}
QPushButton{
	
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius:7px;

}
QPushButton:hover{
	padding-top:5px;
	padding-left:1px;
}</string>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page_Register">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLineEdit" name="lineEdit_R_accuont">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>账号：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_R_password_1">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Normal</enum>
                  </property>
                  <property name="placeholderText">
                   <string>密码：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_R_password_2">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>确认密码：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_R_sure">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>注册</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_Login">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLineEdit" name="lineEdit_L_account">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>账号：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_L_password">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                  <property name="placeholderText">
                   <string>密码：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_L_sure">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>确认</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border:none;
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_Login">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/icons/icons/login.png</normaloff>:/icons/icons/login.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Register">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/icons/icons/register.png</normaloff>:/icons/icons/register.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_error"/>
           <widget class="QWidget" name="page_2"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton{
	border:none;
}
#pushButton:hover{
	padding-top:5px;
	padding-left:5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icons/icons/close.png</normaloff>:/icons/icons/close.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>841</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
